.github/generate-native-matrix.sh

```diff
@@ -0,0 +1,29 @@
+#! /bin/bash
+
+IFS=$'\n'
+modules=($(find . -name pom.xml | sed -E 's@/pom.xml@@g' | sed -E 's@^\./@@g' | grep -vE '^\.$' | sort))
```

Comment:
This does the same:
```
find */ -name pom.xml -printf "%h\n" | sort
```
But both yield this:
```
kafka-streams-quickstart
kafka-streams-quickstart/aggregator
kafka-streams-quickstart/producer
```
(same for `funqy-quickstarts`)
I don't think it makes sense to execute those aggregators individually.
This should work:
```
grep -LR --include=pom.xml '<packaging>pom</packaging>' */ | xargs dirname | sort
```

Reply:
PS: Using `-f` would allow keeping the `/pom.xml` suffix (no `dirname`, `-printf` or `sed` needed).

Reply:
Done.